packages/server-dev/lib/RestartingPage.js

```diff
@@ -0,0 +1,46 @@
+/*
+  Copyright 2020-2022 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import React from 'react';
+
+const RestartingPage = () => {
+  return (
+    <div
+      style={{
+        height: '100vh',
+        textAlign: 'center',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
+    >
+      <div
+        style={{
+          verticalAlign: 'middle',
+          display: 'inline-block',
+        }}
+      >
+        <h3>Rebuilding Lowdefy App</h3>
+        <p>
+          The server is reconfiguring Lowdefy types and restarting. The page will reload when ready.
```

Comment:
Nice! This is a good idea.

This could also happen when app config changes, or .env changes, so maybe `reconfiguring Lowdefy types` is not the right message.

Maybe we could do something like `The server is restarting because your configuration changed. The page will reload when ready.`

As an idea, maybe one day we can display some Lowdefy tips and tricks here, like games do with loading screens.

Reply:
Cool. Changed the message. Yeah, like the tips idea. Would like to fetch that from the telemetry server. Please add to ideas channel.